LLama/Native/NativeApi.cs

```diff
@@ -18,11 +18,6 @@ public static void llama_empty_call()
             llama_max_devices();
         }
 
-        /// <summary>
-        /// Call once at the end of the program - currently only used for MPI
-        /// </summary>
-        public static extern void llama_backend_free();
```

Comment:
Please keep this (but feel free to make it private and suppress warnings about it being unused). When updating the binaries I try keep it one-to-one with llama.cpp, with nothing missing.